src/Autosuggest.js

```diff
@@ -156,6 +157,18 @@ class Autosuggest extends Component {
     }
   }
 
+  maybeSelectFirstSuggestion(event, value) {
+    const { selectFirstSuggestion, multiSection, updateFocusedSuggestion } = this.props;
+
+    if (selectFirstSuggestion) {
+      let sectionIndex = multiSection ? 0 : null;
+      let suggestionIndex = 0;
+
+      updateFocusedSuggestion(sectionIndex, suggestionIndex);
+      this.maybeCallOnChange(event, value, 'auto');
```

Comment:
I think we should trigger `onChange` only when the input value changes, which is not the case here (I agree, it's more similar to "on hover" behaviour, which doesn't change the input value). So, I suggest we don't call `this.maybeCallOnChange()` here.